third_party/python.bzl

```diff
@@ -76,6 +76,17 @@ def tensorboard_python_workspace():
         build_file = str(Label("//third_party:werkzeug.BUILD")),
     )
 
+    http_archive(
```

Comment:
Maybe worth a note here about why we will want to stay on 1.0.0 (until we drop python 2.7 support and this goes away).

Reply:
Good point; will do.